views/admin/index.erb

```diff
@@ -52,4 +52,13 @@
     <li><a href="/change-password">Change Password</a></li>
     <li><a href="/multifactor-manage">Manage Multifactor Authentication</a></li>
   </ul>
+
+  <% if @annotations.any? %>
+    <h2>Annotations</h2>
+    <ul id="annotations-list">
```

Comment:
Probably better to have a table display than list display, so you can easily see when, and link directly to related objects (if support is added for them).